content/docs/reactivesearch/react-native-searchbox/searchbox.md

```diff
@@ -185,6 +185,16 @@ Here, we are specifying that the suggestions should update whenever one of the b
 />
 ```
 
+-   **enablePredictiveSuggestions** `bool` [optional]
```

Comment:
@anik-ghosh-au7 Move it to `Customize Autosuggestions` section.
https://github.com/appbaseio/Docs/blob/3ed42d99f6c1a73f7852bee5f85416bde67815f5/content/docs/reactivesearch/react-native-searchbox/searchbox.md#to-customize-the-autosuggestions